cms/templates/export.html

```diff
@@ -62,18 +64,18 @@ <h1 class="page-header">
           <h2 class="title">${_("About Exporting Libraries")}</h2>
           <div class="copy">
               ## Translators: ".tar.gz" is a file extension, and should not be translated
-              <p>${_("You can export libraries and edit them outside of {studio_name}. The exported file is a .tar.gz file (that is, a .tar file compressed with GNU Zip) that contains the library structure and content. You can also re-import libraries that you've exported.").format(
+              <p>${Text(_("You can export libraries and edit them outside of {studio_name}. The exported file is a .tar.gz file (that is, a .tar file compressed with GNU Zip) that contains the library structure and content. You can also re-import libraries that you've exported.").format(
```

Comment:
You don't need `Text()` unless you use `HTML()`.  It won't hurt anything, but I think it is more clear to leave it out unless it is needed.